QEfficient/exporter/export_utils.py

```diff
@@ -403,11 +403,11 @@ def compile_kv_model_on_cloud_ai_100(
         command.append("-aic-enable-depth-first")
     if len(device_group) > 1:
         mdp_ts_config = {
-            "connections": [{"devices": device_group, "type": "p2p"}],
+            "connections": [{"devices": list(range(len(device_group))), "type": "p2p"}],
```

Comment:
Will it work if I give device_group [0,4]?
Then also we need to compile for 2 devices and deploy on dev 0 and dev 4

Reply:
Yes, it does work, this is tested. [0,4] will be used while execution.